service_management/azure/test/integration/service_bus/queues_test.rb

```diff
@@ -14,154 +14,190 @@
 #--------------------------------------------------------------------------
 require 'integration/test_helper'
 
-describe 'ServiceBus Queues' do
-
+describe Azure::ServiceBus::ServiceBusService do
   subject { Azure::ServiceBus::ServiceBusService.new }
-  let(:queue_name) { ServiceBusQueueNameHelper.name }
-  let(:name_alternative) { ServiceBusQueueNameHelper.name }
+  let(:queue_name) { 'test-queue' }
+  let(:name_alternative) { 'test-queue-alternative' }
   let(:description) {{
-    :default_message_time_to_live => 'P10675199DT2H48M5.4775807S',
-    :duplicate_detection_history_time_window => 'PT10M',
-    :dead_lettering_on_message_expiration => "false",
-    :lock_duration => 'PT30S',
-    :max_delivery_count => "10",
-    :max_size_in_megabytes => "1",
-    :requires_duplicate_detection => "true",
-    :requires_session => "false"
+      :default_message_time_to_live => 'P10675199DT2H48M5.4775807S',
+      :duplicate_detection_history_time_window => 'PT10M',
+      :dead_lettering_on_message_expiration => "false",
+      :lock_duration => 'PT30S',
+      :max_delivery_count => "10",
+      :max_size_in_megabytes => "1",
+      :requires_duplicate_detection => "true",
+      :requires_session => "false"
   }}
   let(:description_alternative) {{
-    :lock_duration => 'PT30S',
-    :max_size_in_megabytes => 2048,
-    :requires_duplicate_detection => true,
-    :requires_session => true,
-    :default_message_time_to_live => 'PT30M',
-    :dead_lettering_on_message_expiration => true,
-    :duplicate_detection_history_time_window => 'PT20M',
-    :max_delivery_count => 20,
-    :enable_batched_operations => true
+      :lock_duration => 'PT30S',
+      :max_size_in_megabytes => 2048,
+      :requires_duplicate_detection => true,
+      :requires_session => true,
+      :default_message_time_to_live => 'PT30M',
+      :dead_lettering_on_message_expiration => true,
+      :duplicate_detection_history_time_window => 'PT20M',
+      :max_delivery_count => 20,
+      :enable_batched_operations => true
   }}
 
-  after { ServiceBusQueueNameHelper.clean }
+  describe 'Service bus queue' do
+    before do
+      VCR.insert_cassette "service_bus/#{name}"
+    end
 
-  it "should be able to create a new queue from a string" do
-    queue = subject.create_queue queue_name
-    queue.must_be_kind_of Azure::ServiceBus::Queue
-    queue.name.must_equal queue_name
-  end
+    after do
+      subject.delete_queue queue_name
```

Comment:
Should we add a check for nil here and for other deletes? 
response = subject.delete_queue queue_name
response.must_equal nil